features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/SnapshotProducer.kt

```diff
@@ -62,6 +65,13 @@ internal class SnapshotProducer(
                 }
             }
         }
+        // in case the view is a WebView and the RUM native view was changed in the meantime,
+        // we need to trigger a full snapshot of the web page through the browser sdk for the
+        // player to have a starting point from which to apply the browser mutation when
+        // required to play the new native RUM view
+        if (view is WebView) {
```

Comment:
should we add the check that `session-replay-webview` feature is enabled here? Or it is independent from the Webview integration?